src/source/reference/get-regions.rst

```diff
@@ -36,7 +36,6 @@ Errors
 ====================    ====    =======================     ==============================================================================
 Field                   Code    Error message               Description
 ====================    ====    =======================     ==============================================================================
-regionIds               2000    missing argument            The parameter needs to be present
```

Comment:
Why is this removed? on the GET /regions I think the regionIds is quite mandatory